packages/react/src/components/RuleBuilder/RuleBuilderHeader.jsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Button } from 'carbon-components-react';
+import { Add32, TextNewLine32 } from '@carbon/icons-react';
+import PropTypes from 'prop-types';
+
+import { settings } from '../../constants/Settings';
+
+import GroupLogic from './GroupLogic';
+import { GroupLogicPropType } from './Rule';
+
+const { iotPrefix } = settings;
+
+const defaultProps = {
+  i18n: {
+    addRule: 'Add rule',
+    addGroup: 'Add group',
+  },
+  groupLogic: 'ALL',
+  testID: 'rule-builder-header',
+};
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  groupLogic: GroupLogicPropType,
```

Comment:
if we could get some comments for some of the less obvious props as to what they do

Reply:
resolved with 7fac216